.github/workflows/docker-image.yml

```diff
@@ -39,5 +39,9 @@ jobs:
       uses: docker/build-push-action@v5
       with:
         push: ${{ github.event_name != 'pull_request' }}
-        tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+        tags: |
+          ${{ steps.meta.outputs.tags }}
+          # Add 'latest' tag for master branch
+          ${{ if(github.ref == 'refs/heads/master' && github.event_name == 'push') }} 'latest' ${{ else }} '' ${{ endif }}
+
```

Comment:
Rather than editing the tags here, I'd recommend configuring the docker metadata action above, instead.

[by default, if you specify](https://github.com/docker/metadata-action?tab=readme-ov-file#latest-tag) something like
```
flavor: latest=auto
tags: |
  type=schedule
  type=ref,event=branch
  type=ref,event=tag
  type=ref,event=pr
```
and then applied a "git tag" of `v1.2.3`, it would apply the image labels `v1.2.3` and `latest`

This just makes releasing a new `latest` tag slightly more deliberate, since the hammers will auto-update at all sites, we want to avoid accidental releases.